components/admin_console/admin_definition.jsx

```diff
@@ -5589,35 +5569,13 @@ const AdminDefinition = {
                         isDisabled: it.not(it.userHasWritePermissionOnResource('experimental')),
                     },
                     {
-                        type: Constants.SettingsTypes.TYPE_DROPDOWN,
-                        key: 'ServiceSettings.ExperimentalChannelSidebarOrganization',
-                        label: t('admin.experimental.experimentalChannelSidebarOrganization.title'),
-                        label_default: 'Experimental Sidebar Features',
-                        help_text: t('admin.experimental.experimentalChannelSidebarOrganization.desc'),
-                        help_text_default: 'When enabled, users can access experimental channel sidebar features, including collapsible sections and unreads filtering. If default on, this enabled the new sidebar features by default for all users on this server. Users can disable the features in **Account Settings > Sidebar > Experimental Sidebar Features**. If default off, users must enable the experimental sidebar features in Account Settings. [Learn more](!https://about.mattermost.com/default-sidebar/) or [give us feedback](!https://about.mattermost.com/default-sidebar-survey/)',
+                        type: Constants.SettingsTypes.TYPE_BOOL,
+                        key: 'ServiceSettings.EnableLegacySidebar',
+                        label: t('admin.experimental.enableLegacySidebar.title'),
+                        label_default: 'Enable Legacy Sidebar',
+                        help_text: t('admin.experimental.enableLegacySidebar.desc'),
+                        help_text_default: 'When enabled, users cannot access new sidebar features including custom, collapsible categories and unread channel filtering. We recommend only enabling the legacy sidebar if users are experiencing breaking changes or bugs.',
```

Comment:
Not to be totally pedantic, but this change is technically a breaking change, so they're guaranteed to experience at least one. We might want to remove that part of the help text.

Reply:
I think the point is that this should only be turned on if the new sidebar is broken. It's not necessarily a breaking change for all users, only some.

Reply:
I guess there's just some ambiguity about what constitutes a breaking change. I'm looking at it that something major is changing and some old functionality isn't available because of that, so that's my thinking there. That said, that doesn't particularly matter.